docs/Windows.md

```diff
@@ -3,25 +3,55 @@
 These instructions cover building 32blit on Windows (without WSL).
 
 - [Prerequisites](#prerequisites)
-- [Using Visual Studio](#using-visual-studio)
+- [Setting Up](#setting-up)
 - [Building & Running on 32Blit](#building--running-on-32blit)
 - [Building & Running Locally](#building--running-locally)
   - [Build Everything](#build-everything)
+- [Troubleshooting](#troubleshooting)
+  - [Multiple definition of `cleanup_glue`, `_reclaim_reent` etc](#multiple-definition-of-cleanup_glue-_reclaim_reent-etc)
+
+To build using Visual Studio [see here](Windows-VisualStudio.md). The rest of these instructions cover command-line builds.
```

Comment:
Hmm, I had that after the prerequisites section because the VS docs don't mention installing python/the 32blit tools (and git?).